doc/python/dropdowns.md

```diff
@@ -444,5 +444,101 @@ fig.update_layout(title_text="Yahoo")
 fig.show()
 ```
 
+### Creating several independent graphs and using Jinja to insert them into a JavaScript enabled webpage 
```

Comment:
```suggestion
### Embedding Multiple Graphs in a Webpage using Jinja
```
We can shorten this if possible, so it takes up less space on the left sidebar

Reply:
Good suggestion.  What would you think of "Using a Dropdown to Select a Graph Using Jinja" as a short title?